aws/logs_monitoring/tools/add_new_region.sh

```diff
@@ -12,7 +12,7 @@ set -e
 
 OLD_REGION='us-east-1'
 
-PYTHON_VERSIONS_FOR_AWS_CLI=("python3.9")
+PYTHON_VERSIONS_FOR_AWS_CLI=("python3.11")
```

Comment:
do we support only one version here ? why an array then ?

Reply:
it seems we support only the highest version, not sure why